packages/eslint-plugin-template/docs/rules/no-restricted-syntax.md

```diff
@@ -0,0 +1,63 @@
+<!--
+
+  DO NOT EDIT.
+
+  This markdown file was autogenerated using a mixture of the following files as the source of truth for its data:
+  - ../../src/rules/no-restricted-syntax.ts
+  - ../../tests/rules/no-restricted-syntax/cases.ts
+
+  In order to update this file, it is therefore those files which need to be updated, as well as potentially the generator script:
+  - ../../../../tools/scripts/generate-rule-docs.ts
+
+-->
+
+# `@angular-eslint/template/no-restricted-syntax`
+
+Disallows specified syntax
+
+- Type: suggestion
+- Category: Stylistic Issues
+
+<br>
+
+## Rule Options
+
+The rule does not have any configuration options.
```

Comment:
Unfortunately, [there's a bug on `json-schema-to-typescript` so it doesn't compile schema well if it contains `oneOf`](https://github.com/bcherny/json-schema-to-typescript/issues/381).